mParticle-Appboy/MPKitAppboy.m

```diff
@@ -280,6 +280,13 @@ - (void)start {
           optionsDictionary[ABKSDKFlavorKey] = @(MPARTICLE);
 #pragma clang diagnostic pop
         }
+        
+        optionsDictionary[ABKDisableAutomaticLocationCollectionKey] = @(NO);
+        if (self.configuration[@"ABKDisableAutomaticLocationCollectionKey"]) {
+            if ([self.configuration[@"ABKDisableAutomaticLocationCollectionKey"] caseInsensitiveCompare:@"true"] == NSOrderedSame) {
```

Comment:
if the setting is `False`, I think we should be explicitly setting it to false, kinda like forwardScreenviews above.

Reply:
@samdozor  but it's already set to false on line 284 above.

Reply:
ah! sorry. got it.